posthog/settings/web.py

```diff
@@ -60,6 +60,9 @@
 # if `true` we disable session replay if over quota
 DECIDE_SESSION_REPLAY_QUOTA_CHECK = get_from_env("DECIDE_SESSION_REPLAY_QUOTA_CHECK", False, type_cast=str_to_bool)
 
+# if `true` we disable feature flags if over quota
+DECIDE_FEATURE_FLAG_QUOTA_CHECK = get_from_env("DECIDE_FEATURE_FLAG_QUOTA_CHECK", False, type_cast=str_to_bool)
```

Comment:
this could be an array of team IDs to either explicit limit or explicitly not limit.

Reply:
Just curious, why wouldn't we control this with our own feature flag?

Reply:
I guess we could!  I was just following the pattern of the other environment variable controls.  We've generally avoided using feature flags that are evaluated within `/decide` itself, though (this variable is passed into `get_decide`.

Reply:
> We've generally avoided using feature flags that are evaluated within `/decide` itself

Ha! I guess that could get recursive. I'm not suggesting making this change. Just curious.